home/migrations/0084_alter_contentpage_whatsapp_body.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.11 on 2024-11-19 09:08
+
+
+from wagtail.blocks.migrations.migrate_operation import MigrateStreamData
+from wagtail.blocks.migrations.operations import BaseBlockOperation
+from wagtail.blocks.migrations.utils import formatted_list_child_generator
+from django.db import migrations
+
+
+def list_string_to_action(block_value):
+    # formatted_list_child_generator requires at least one item
+    if not block_value:
+        return []
+    new_list = []
+    for list_item in formatted_list_child_generator(block_value):
```

Comment:
These items will now be in streamfield format, not list format, so I think we'll have issues running it through twice. I think this only works now because all the values were set to `null` from the previous migration. Maybe we just have:
```
if not block_value:
    return []
return block value
```
To set any null values to an empty list?